js/gateway-api/gatewayApi.js

```diff
@@ -120,9 +120,12 @@ export const hentApiUrl = () => {
     if (url.indexOf('tjenester.nav') > -1) {
         // Prod
         return 'https://syfoapi.nav.no/syfosoknad/api';
-    } else if (url.indexOf('localhost') > -1 || url.indexOf('herokuapp') > -1) {
-        // Lokalt
+    } else if (url.indexOf('herokuapp') > -1) {
+        // Heroku
         return '/syfoapi/syfosoknad/api';
+    } else if (url.indexOf('localhost') > -1) {
```

Comment:
dette blir kanskje køddent om man går mot lokal mock, men er jo bare å kjøre mot docker compose oppsettet uansett ?

Reply:
mhm, dog det er noen tilfeller hvor det er greit å ha lokal mock.. f.eks. i tilfeller hvor man starter på ny dialog, så er man ikke avhengig av at syfosoknad produserer greier; man kan bare anta at strukturen er sånn og sånn 🤷‍♀

Reply:
kan vi f.eks. bruke et flagg i `.env` for å bestemme videre om vi skal bruke lokal mock eller kjøre mot docker-compose?

Reply:
ja, må bare få propagert det inn i app koden, skal finne en løsning